.eslintrc.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     'ecmaVersion': 2018
   },
   'rules': {
-    'semi': ['error', 'always']
+    'semi': ['error', 'always'],
+    'camelcase': 'off'
```

Comment:
Can you please explain why this was necessary? Do any of the new additions require this?

Reply:
The necessity arose because the validation logic has to read keys and values of translation entries from PO files. One of those keys - `msgid_plural` - is written in snake_case. I was considering disabling the rule by adding comments where necessary, but because that occurs in a number of places I decided to go with an update in `.eslintrc.js` file. Should I solve it by adding comments wherever needed?